docs/notifications.md

```
@@ -75,6 +75,13 @@ In addition to direct access of data (labels and annotations) stored as KV, ther
 | Names | - | []string | Returns the names of the label names in the LabelSet. |
 | Values | - | []string | Returns a list of the values in the LabelSet. |
 
+`Remove` expects a slice of strings. To remove literal keys in a template, use
+the `stringSlice` function:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)